Automattic/jetpack
# Beta Plugin: Prevent Updating Stable Version in Docker Instance

Fixes #18496

#### Changes proposed in this Pull Request:

* When running the Jetpack Beta plugin in Docker, this prevents the prompt to update the stable version of the plugin, which would override the local development environment in `projects/plugins/jetpack`.

* After doing some digging and a lot of testing, it looks like the beta plugin is configured to not auto-update the stable version anyway as far as I can see.

#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?



Nope

#### Testing instructions:






* Spin up a local site in a Docker instance
* Go to the Jetpack Beta Plugin Settings
* See the prompt to update the latest stable version (**NOTE** - do not click this button, it will break things)
* Switch to this branch
* Visit the Jetpack Beta Plugin settings again
* See that the prompt is gone 
* Everything else should work fine.

To test this further you can:

* Spin up a Jurassic Ninja Site with the Beta plugin
* Switch to this branch.
* Edit the plugin files in WP-Admin and change stable Jetpack's version number to something else (e.g. 5.0)
* You should see the prompt to update to latest stable
* Update should work 

**Before (in docker):**

![Screen Shot on 2021-01-28 at 17:24:27](https://user-images.githubusercontent.com/33553323/106206210-afa45e80-618d-11eb-8f8b-8740ffa176db.png)

**After (in docker):**

![what we shoudl see](https://user-images.githubusercontent.com/33553323/106206063-7c61cf80-618d-11eb-8296-86f631f86bf7.png)



#### Proposed changelog entry for your changes:


* Prevents updating stable version of Jetpack when using beta plugin in Docker instance.
